### DISCUSSION TITLE
Are durable entities supported?

### DISCUSSION THREAD

**Discussion by tomkerkhove** [scale-tone/DurableFunctionsMonitor#114] [Q&A] [answered]
I was wondering if I can explore my durable entities with Durable Functions Monitor?

**Comment by scale-tone** [comment, marked as answer]
> I was wondering if I can explore my durable entities with Durable Functions Monitor?

Absolutely yes :)
They should appear in the list, just like orchestrations. And you can also filter for them:
![image](https://user-images.githubusercontent.com/5447190/128348611-1effd765-3ba6-4ee8-97ce-89e4db85b477.png)

**Reply by tomkerkhove** [reply]
I have just found them - Great, thank you!
